rwgame/RWGame.cpp

```diff
@@ -400,17 +403,27 @@ int RWGame::run() {
         }
         RW_PROFILE_END();
 
-        auto now = clock.now();
-        float timer =
-            std::chrono::duration<float>(now - last_clock_time).count();
-        last_clock_time = now;
-        accum += timer * timescale;
+        time_physics_curr = std::chrono::steady_clock::now();
+
+        deltaPhysics = std::chrono::duration<float>(time_physics_curr - time_physics_prev).count();
+
+        world->dynamicsWorld->stepSimulation(deltaPhysics);
+        time_physics_prev = time_physics_curr;
+
+        deltaGame = std::chrono::duration<float>(std::chrono::steady_clock::now() - time_gameclock).count();
```

Comment:
It looks like timescale has been removed. We need some way to control the "real" speed for the slow-motion effects.

Reply:
I'll try how it works with:
deltaGame -= GAME_TIMESTEP * timescale;
time_gameclock += std::chrono::duration<float>(GAME_TIMESTEP) * timescale;